crypto/engine/eng_pkey.c

```diff
@@ -57,6 +57,7 @@ EVP_PKEY *ENGINE_load_private_key(ENGINE *e, const char *key_id,
                                   UI_METHOD *ui_method, void *callback_data)
 {
     EVP_PKEY *pkey;
+    bool copied_key;
```

Comment:
Test failures are relevant

`bool` type, `true`, and `false` are not supported in C89, AFAIR (and openssl doesn't correctly build with your patch). So could you please change it to some integer type?

Reply:
Thank you, I changed it to an `int`.